python/ray/serve/_private/replica.py

```diff
@@ -579,7 +544,12 @@ def user_health_check():
         self.autoscaling_metrics_store = InMemoryMetricsStore()
         self.metrics_pusher = MetricsPusher()
         if autoscaling_config:
-            process_remote_func = controller_handle.record_autoscaling_metrics.remote
+            self.controller_handle = ray.get_actor(
```

Comment:
Out of curiosity, why push the `ray.get_actor()` call down to here?

Reply:
`controller_handle` is used only for this purpose, so just pushing it down to avoid bloating the number of arguments getting passed around